modules/libcom/src/error/errlog.c

```diff
@@ -680,7 +680,7 @@ static void errlogThread(void)
             print->pos = 0u;
 
             if(nLost && console)
-                fprintf(console, "errlog: lost %zu messages\n", nLost);
+                fprintf(console, "errlog: lost %lu messages\n", (unsigned long) nLost);
```

Comment:
I see that you replaced all `%zu` specifiers to `%lu`. Is the `z` modifier unsupported on some compilers?